src/App.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const arrNums = getNumbers(1, 42);
```

Comment:
Make this constant outside the component so that the variable is not re-rendered every time it is rendered. To 5 line for example